src/main/java/Event.java

```diff
@@ -0,0 +1,38 @@
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    // Todo: Handle exceptional cases
+    public static Task createTask(String[] commandArgs) throws IllegalArgumentException {
+        String description = "";
+        String at = "";
+
+        int toggle = 0;
+        for (int i = 0; i < commandArgs.length; i++) {
+            if (commandArgs[i].equals("/at")) {
+                toggle = 1;
+                continue;
+            }
+            // current token is part of description
+            if (toggle == 0) {
+                description += commandArgs[i] + " ";
+            }
```

Comment:
the "else if" statement should be on the same line as the previous curly bracket :)